File: 00_Learn_The_Basics/Things_To_Know_In_A_Language/Switch_Case/gradesystem.cpp
//Showing the choice of the weekday using the switch cases 

#include<iostream>
using namespace std;

int main()
{
        char g;

        cout<<"Enter the grade (A,B,C,D,F) "<<endl;;
        cin>>g;

        switch(g)
        {
        case 'A':
            cout<<"Excellent Performance "; 
            break;

        case 'B':
            cout<<"Very Good Performance";  
            break;
        
         case 'C':
            cout<<"Good Performance";  
            break;

        case 'D':
            cout<<"Average Performance ";  
            break;

        case 'F':
            cout<<"Fail ";  
            break;

        default:
            cout<<"Invalid Grade ";
        
    }

 }

File: 00_Learn_The_Basics/Things_To_Know_In_A_Language/Switch_Case/GradeSystem.java
//Showing the choice of the weekday using the switch cases 

import java.util.*;

public class GradeSystem
{
    public static void main(String[] args)
    {
        char g;

        Scanner in=new Scanner(System.in);


        System.out.println("Enter the grade (A,B,C,D,F) ");
        g=in.next().charAt(0);

        switch(g)
        {
        case 'A':
            System.out.println("Excellent Performance "); 
            break;

        case 'B':
            System.out.println("Very Good Performance");  
            break;
        
         case 'C':
            System.out.println("Good Performance");  
            break;

        case 'D':
            System.out.println("Average Performance ");  
            break;

        case 'F':
            System.out.println("Fail ");  
            break;

        default:
            System.out.println("Invalid Grade ");
        
    }

 }
}
